client/utils/constants.js

```diff
@@ -0,0 +1,22 @@
+export const TABLE_SORT_ORDERS = {
+    ASC: 'ASCENDING',
+    DESC: 'DESCENDING'
+};
+
+export const DATA_MANAGEMENT_TABLE_SORT_OPTIONS = {
+    NAME: 'NAME',
+    ATS: 'ATS',
+    PHASE: 'PHASE'
+};
+
+export const TEST_PLAN_VERSION_PHASES = {
+    RD: 'RD',
+    DRAFT: 'DRAFT',
+    CANDIDATE: 'CANDIDATE',
+    RECOMMENDED: 'RECOMMENDED'
+};
+
+export const DATA_MANAGEMENT_TABLE_FILTER_OPTIONS = {
+    ALL: 'ALL',
+    ...TEST_PLAN_VERSION_PHASES
+};
```

Comment:
I'm down with using enums like this. I do think there's a place on the server that uses this same pattern. Since this is only used in the sorting, I wonder what you think about moving these constants to the data management hooks file or somewhere like that (maybe the data management utils.js file?) until they're needed in multiple places?

Reply:
I tried to localize each of them to where they are most heavily used. One in specific can and is used in many places, `TEST_PLAN_VERSION_PHASES` so I kept that in this more global file.